src/main/java/org/osiam/resources/controller/ServiceProviderConfigController.java

```diff
@@ -23,20 +23,20 @@
 
 package org.osiam.resources.controller;
 
-import java.util.HashSet;
-import java.util.Set;
-
-import org.osiam.resources.helper.RequestParamHelper;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
 
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.HashSet;
+import java.util.Set;
 
 @Controller
```

Comment:
You did it for `GroupController`, too, and it will definitively work:

> public @interface RestController
> A convenience annotation that is itself annotated with @Controller and @ResponseBody.
> Types that carry this annotation are treated as controllers where @RequestMapping methods assume @ResponseBody semantics by default.